tests/MongoDB.Bson.Tests/Serialization/Conventions/StringIdStoredAsObjectIdConventionTests.cs

```diff
@@ -25,76 +25,58 @@ namespace MongoDB.Bson.Tests.Serialization.Conventions
 {
     public class StringIdStoredAsObjectIdConventionTests
     {
+        private readonly IBsonSerializer _defaultInt32Serializer = BsonSerializer.LookupSerializer(typeof(int));
+        private readonly IBsonSerializer _defaultStringSerializer = BsonSerializer.LookupSerializer(typeof(string));
+
         [Fact]
         public void Apply_should_ignore_any_member_that_is_not_the_id()
```

Comment:
wrong tests order?

Reply:
We haven't necessarily been consistent in the past... but the best order for the tests is to match as closely as possible the order of the code being tested. That makes it easier to verify that all the code is being tested.

This test method is first because it is testing the first three lines of the Apply method.